recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.0.bb

```diff
@@ -0,0 +1,40 @@
+SUMMARY = "Linux kernel modules for Nvidia Display"
+DESCRIPTION = "${SUMMARY}"
+LICENSE = "Proprietary"
```

Comment:
The license described in the COPYING file is `MIT | GPLv2`

Reply:
I will update the `LICENSE` list